### PR TITLE
Add Connect to USB article

### PR DESCRIPTION
**Pull request by mattwojo**

### CODE REVIEW

File: WSL/connect-usb.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Connect USB devices to WSL 2
```
Comment (craigloewen-msft) [inline WSL/connect-usb.md:2]:
```suggestion
title: Connect USB devices
```

Suggesting this edit so we can try and move away from "WSL 2" and the  norm can just become "WSL" :)

File: WSL/connect-usb.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+title: Connect USB devices to WSL 2 
+description: Learn how to connect a USB device to your WSL 2 Linux distribution using usbipd-win.
+ms.date: 11/09/2021
+ms.topic: article
+ms.localizationpriority: medium
+---
+
+# Connect USB devices to WSL 2
+
+This guide will walk through the steps necessary to connect a USB device to a Linux distribution running on WSL 2 using the USB/IP open-source project, [usbipd-win](https://github.com/dorssel/usbipd-win).
```
Comment (craigloewen-msft) [inline WSL/connect-usb.md:11]:
```suggestion
This guide will walk through the steps necessary to connect a USB device to a Linux distribution running on WSL using the USB/IP open-source project, [usbipd-win](https://github.com/dorssel/usbipd-win).
```

File: WSL/connect-usb.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+## Prerequisites
+
+- Running on either Windows 11 (Build 22000 or higher) or Windows 10 (Build ?? or higher).
+- Running [Linux kernel 5.10.60.1 or higher](./kernel-release-notes.md).
+
+> [!NOTE]
+> To check your Windows version and build number, select **Windows logo key + R**, type **winver**, select **OK**. You can update to the latest Windows version by selecting **Start** > **Settings** > **Windows Update** > **[Check for updates](ms-settings:windowsupdate)**.
+> To check your Linux kernel version, open your Linux distribution and enter the command: `uname -a`. To manually update to the latest kernel, open PowerShell and enter the command: 'wsl --update`.
+
+## Install the USBIP project
```
Comment (craigloewen-msft) [inline WSL/connect-usb.md:24]:
```suggestion
## Install the USBIPD-WIN project
```

Trying to give a bit more of a shoutout to this exact project

File: WSL/connect-usb.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+> [!NOTE]
+> Alternatively, you can also install the usbipd-win project using [Windows Package Manager](/windows/package-manager/winget/) (winget). If you have already installed winget, just use the command: `winget install --interactive --exact dorssel.usbipd-win` to install usbipd-win. If you leave out --interactive, winget may immediately restart your computer if that is required to install the drivers.
+
+This will install:
+
+- A service called `usbipd` (display name: USBIP Device Host). You can check the status of this service using the Services app from Windows.
+- A command line tool `usbipd`. The location of this tool will be added to the PATH environment variable.
+- A firewall rule called `usbipd` to allow all local subnets to connect to the service. You can modify this firewall rule to fine tune access control.
+
+## Install the USBIP tools and hardware database
```
Comment (craigloewen-msft) [inline WSL/connect-usb.md:41]:
```suggestion
## Install the USBIP tools and hardware database in Linux
```

Feel free to take or not take this suggestion, wondering if making it a bit clearer that these steps are done in WSL is valuable.

**Review by craigloewen-msft — APPROVED** [review]

File: WSL/connect-usb.md (hunk 2 above)
Comment (mattwojo) [reply WSL/connect-usb.md:11]: Based on our discussion yesterday, deleting this change as USB support is only for WSL 2 (and won't work for WSL 1 from what I understand).

**Pull request closed without merging**